api/api.go

```diff
@@ -18,6 +18,7 @@ import (
 const (
 	defaultLinesOfContext uint = 2
 	maxLinesOfContext     uint = 20
+	maxLimit              int  = 100000
```

Comment:
Out of curiosity, is this number informed by anything or is it just reasonably large?